File: Tests/WebApiController/ClientApp/src/app/get-complex/models/get-complex-model.ts
/* eslint-disable */
// tslint:disable

import { GetComplexModelService } from "./get-complex-model-service";

export class GetComplexModel {
    public property?: string;
    public service?: GetComplexModelService;
    public services?: GetComplexModelService[];

    public constructor(init?: Partial<GetComplexModel>) {
        Object.assign(this, init);
    }
}

// outputid:627408ca-a818-4326-b843-415f5bbfb028

File: Tests/AspDotNetAnnotations/Models/GetComplexModel.cs
using System.Collections.Generic;

namespace AspDotNetAnnotations.Models
{
    public class GetComplexModel
    {
        public string Property { get; set; } = "";

        public GetComplexModelService Service { get; set; } = new();
        public List<GetComplexModelService> Services { get; set; } = [];
    }

    public class GetComplexModelService
    {
        public string Property { get; set; } = "";
    }
}